.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Maybe want an exclude for this? I use goland and exclude my `.idea` folder.

Reply:
Left it in so others can use debug configurations out of the box, but I don't have a strong opinion either way